### ISSUE TITLE
There is not information about rebate in product data on the basket page

### ISSUE THREAD

**Issue by akropivko** [aimeos/ai-client-html#208]
Version - 2023.07
Integration - Laravel
System - Windows10

There is a rebate value on the backend:
![Screenshot_40](https://github.com/aimeos/ai-client-html/assets/66216139/439d4d85-fac4-411c-a1aa-2a792f687050)

There is a rebate value on the catalog list:
![Screenshot_41](https://github.com/aimeos/ai-client-html/assets/66216139/7a72257c-849c-4282-8497-391ac411be86)

There is a rebate value the catalog detail page:
![Screenshot_42](https://github.com/aimeos/ai-client-html/assets/66216139/cab2ea64-3cd7-404a-a964-5df32ac701ee)

But there is not rebate value  on the basket page (/shop/basket), dump ($product) 
![Screenshot_44](https://github.com/aimeos/ai-client-html/assets/66216139/1cb366cc-53f5-4f9e-9bfe-bd53474c4099)

**Comment by aimeos** [comment]
That's true, already given product rebates are not shown in the basket any more and the `price.rebate` fields is used for discounts granted by coupon codes in the basket, during the checkout and in the final order.

**Comment by akropivko** [comment]
Thanks for clarification
